loadConfig.d.ts

```diff
@@ -0,0 +1,4 @@
+import type { Config } from './types/config'
+
+declare function loadConfig(path: string): Config
+export = loadConfig
```

Comment:
Does this seem correct to you, @thecrypticace?

Started form the `resolveConfig` file

Reply:
for `require("tailwindcss/loadConfig")` yep looks right to me